src/showTodos.js

```diff
@@ -0,0 +1,54 @@
+const showTodo = (todosList) => {
+  const sortedTodos = todosList.list.sort((a, b) => a.index - b.index);
+  const todosContainer = document.querySelector('.todos');
+  let html = '';
+  sortedTodos.forEach(({ completed, description, id }) => {
+    const checkedTodo = completed ? 'checked' : '';
+    const checkClass = completed ? 'checked' : '';
+    html += `<div class="todo-item">
+        <div>
+        <input id="${id}" class="todo-check" type="checkbox" ${checkedTodo} />
+        <input id="${id}" class="todo-edit ${checkClass}" type="text" value="${description}" />
+        </div>
+        <button id="${id}" class="remove-btn"> <i class="fas fa-trash"></i></button>
+        </div>
+    `;
+  });
+  todosContainer.innerHTML = html;
+
+  const removeBtns = document.querySelectorAll('.remove-btn');
+  removeBtns.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      const element = btn.parentNode;
+      element.remove();
+      todosList.removeTodo(e.target.parentNode.id);
+    });
+  });
+
+  const todosContent = document.querySelectorAll('.todo-edit');
+  todosContent.forEach((todo) => {
+    todo.addEventListener('focusin', (e) => {
+      e.target.parentNode.parentNode.style.background = 'rgb(241,240,204)';
+      e.target.style.background = 'rgb(241,240,204)';
+    });
+    todo.addEventListener('focusout', (e) => {
+      e.target.style.background = 'white';
+      e.target.parentNode.parentNode.style.background = 'white';
```

Comment:
Although, it is not an issue, but from Accessibility point of view it actually creates an issue. Because when you use it . It actually inject the style inline which actually distrub the natural flow of CSS paint. Because `inline-style` have higher priorty than others.